### ISSUE TITLE
Cell shader (Toon)

### ISSUE THREAD

**Issue by n-pigeon** [godotengine/godot#8061]
Would be great if new fixed shader supported cell shader for diffuse (based on current PBR diffuse?) with Softness and Extent (size) parameters :)

PS
Yes, I've played new Zelda... xD

This one I made in Blender, which toon shader and RT shadows are not the best quality :B
![heads_001_r](https://cloud.githubusercontent.com/assets/6152134/24062216/f6370d6c-0b5a-11e7-84c7-7c79a28893b7.png)

**Comment by reduz** [comment]
Hopefully it should come with all built in

**Comment by Chatto** [comment]
What is the difference? I've not done much yet with shaders, but can you give an example of what quality?

**Comment by n-pigeon** [comment]
@Chatto, sorry, can you rephrase your question? I don't understand. :)

**Comment by reduz** [comment]
this was added already, closing
